apps/predbat/predbat.py

```diff
@@ -8557,6 +8656,9 @@ def initialize(self):
             self.record_status("ERROR: Exception raised {}".format(e))
             raise e
 
+        self.config_grouping = self.get_arg("config_grouping", default=list(CONFIG_GROUPS.keys())[0])
```

Comment:
I think the default here should be 'user defined' or similar, this will set the value of config_grouping in HA. The user would then select an option which would update the settings but the dropdown wouldn't change. We don't want the settings to overwrite every time the user changes a value